README.md

```diff
@@ -92,10 +92,36 @@ export default Component.extend({
 });
 ```
 
+## Configure emotion's babel plugin
+
+Many additional options are supported, optionally, via [babel-plugin-emotion][babel-plugin-emotion].  By default, all options are disabled with exception to whitespace trimming (default).
+
+Example configuration:
+
+```js
+/* ember-cli-build.js */
+let app = new EmberApp(defaults, {
+  emotion: {
```

Comment:
What do you think about making these Babel options available  under `emotion.babel` or something? In the future there might be other options, and it might be confusing to mix options for the addon and options for the Babel plugin all together.

Reply:
done